kde-plasma/kscreenlocker/kscreenlocker-9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit kde5
+
+DESCRIPTION="KDE Plasma Screen Locker"
+KEYWORDS=""
+IUSE=""
+
+COMMON_DEPEND="
+	$(add_frameworks_dep kcmutils)
+	$(add_frameworks_dep kcrash)
+	$(add_frameworks_dep kdeclarative)
+	$(add_frameworks_dep kdelibs4support)
+	$(add_frameworks_dep kglobalaccel)
+	$(add_frameworks_dep kidletime)
+	$(add_frameworks_dep plasma)
```

Comment:
lddtree should reveal additional transitive deps, when I looked into it I had found at least kwindowsystem, kxmlgui, ki18n and knotifications.